events.md

```diff
@@ -4,7 +4,18 @@ author_profile: false
 title: Events
 ---
 
-Coming soon: this page will have a list of past and upcoming Spack talks,
-tutorials, webinars, and other events of interest.
+## 2020
 
-**For now, check out our page on [Spack at SC19](/spack-at-sc19/)**.
+### [ISC High Performance](https://www.isc-hpc.com/)
+<small>21-25 June 2020 | Frankfurt, Germany (PENDING)</small>  
+The ISC (formerly known as the International Supercomputing Conference) caters to the hardware and software demands of global research centers and businesses in the fields of HPC, artificial intelligence, machine learning and data analytics.
```

Comment:
Let's not put this one in until we get confirmation of acceptance.

Reply:
Sorry, somehow didn't see these notes until today. Fixed in a21b820